### PR TITLE
feat(PX-3970): top banner for shows tab

### PR DESCRIPTION
**Pull request by kiryl-zubarau**
This PR:

- adds `featured` sort option
- does not filter by status if it's null
- adds `isFeatured` field to the `show` entity
- adds filtering by `isDisplayable` for the `showsConnection`

JIRA - https://artsyproduct.atlassian.net/browse/PX-3970

image

### CODE REVIEW

File: src/schema/v2/partner.ts [hunk 1]
```diff
@@ -562,8 +567,9 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             page,
             size,
             sort: args.sort,
-            status: args.status,
+            status: args.status || undefined,
```
Comment (dzucconi) [inline src/schema/v2/partner.ts:570]: Why can't this be `false`?

File: src/schema/v2/sorts/show_sorts.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ const SHOW_SORTS = {
   END_AT_DESC: { value: "-end_at" },
   FEATURED_ASC: { value: "featured" },
   FEATURED_DESC: { value: "-featured" },
+  FEATURED: { value: "-featured,-end_at" },
```
Comment (dzucconi) [inline src/schema/v2/sorts/show_sorts.ts:8]: Should this be `FEATURED_DESC_END_AT_DESC`?

File: src/schema/v2/partner.ts (hunk 1 above)
Comment (kiryl-zubarau) [reply src/schema/v2/partner.ts:570]: if you mean something like `status: args.status || false` it throws an error
image

Comment (dzucconi) [reply src/schema/v2/partner.ts:570]: That answers my question yeah (I meant why the need to remove the arg vs let it hang as false). Weird though that a boolean flag can be true but not false :(

File: src/schema/v2/sorts/show_sorts.ts (hunk 2 above)
Comment (kiryl-zubarau) [reply src/schema/v2/sorts/show_sorts.ts]: Renamed!

**Pull request merged**